src/helpers/buildWalletSections.js

```diff
@@ -252,8 +252,12 @@ const withBalanceSection = (
   );
   let balanceSectionData = [...assets];
 
-  const totalValue = convertAmountToNativeDisplay(
+  const totalBalanceWithSavingsValue = add(
```

Comment:
should we use useMemo here?

Reply:
hey! this is not a component - but it is using selectors !

Reply:
Damn!  I was convinced that I was looking at the WalletScreen.js file 😅